SR2022/2022-04-26-competition-wrap-up.md

```diff
@@ -0,0 +1,61 @@
+---
+to: Student Robotics 2022
+subject: That's a wrap!
+---
+
+![Competition Photo](TODO)
+
+Hi all,
+
+We'd like to thank you for taking part in Student Robotics 2022 and making it such a successful event! It was fantastic to finally see robots in-person again after two years of being virtual. An article about the competition, results, and game has been [posted on our website](https://studentrobotics.org/news/2022-04-24-sr2022-knockouts/).
+
+Congratulation again to our winners:
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1st Place             | [Hills Road Sixth Form College](https://www.hillsroad.ac.uk/)
+| 2nd Place             | [Queen Mary's College](https://www.qmc.ac.uk/)
+| 3rd Place             | [Brockenhurst College](https://brock.org.uk/)
+| Rookie Award          | [Abingdon School](https://www.abingdon.org.uk/)
+| Committee Award       | [Queen Mary's College](https://www.qmc.ac.uk/)
+| Robot and Team Image  | [Hills Road Sixth Form College](https://www.hillsroad.ac.uk/)
+| Online Presence       | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
```

Comment:
Perhaps something like this would be good to include the TLAs?
```suggestion
| 1st Place             | HRS - [Hills Road Sixth Form College](https://www.hillsroad.ac.uk/)
| 2nd Place             | QMC - [Queen Mary's College](https://www.qmc.ac.uk/)
| 3rd Place             | BRK - [Brockenhurst College](https://brock.org.uk/)
| Rookie Award          | ABS - [Abingdon School](https://www.abingdon.org.uk/)
| Committee Award       | QMC - [Queen Mary's College](https://www.qmc.ac.uk/)
| Robot and Team Image  | HRS - [Hills Road Sixth Form College](https://www.hillsroad.ac.uk/)
| Online Presence       | HAM - [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
```

Reply:
I think for the email it's fine to leave without. I know in the website post we use both TLAs and team names, so perhaps makes sense to put there instead?